src/App.tsx

```diff
@@ -15,36 +15,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+export enum SortKeys {
+  ALPHABETICAL = 'Alphabetical',
+  LENGTH = 'Length',
+}
+
+export const App: FC = () => {
+  const [sort, setSort] = useState<string>('');
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const getSortedList = (): string[] => {
+    let sortedGoods = [...goodsFromServer];
+
+    switch (sort) {
+      case SortKeys.ALPHABETICAL:
+        sortedGoods.sort();
+        break;
+      case SortKeys.LENGTH:
+        sortedGoods.sort((a, b) => a.length - b.length);
+        break;
+      default:
+        sortedGoods = [...goodsFromServer];
+    }
+
+    return isReversed ? sortedGoods.reverse() : sortedGoods;
+  };
+
+  const toggleReverse = (): void => {
+    setIsReversed(prev => !prev);
+  };
+
+  const isOriginalOrder = sort === '' && !isReversed;
```

Comment:
```suggestion
  const isOriginalOrder = !sort && !isReversed;
```
You don't need to compare with an empty string, just check if a value exists or not